src/components/ItaliaTheme/Blocks/VideoGallery/Block/ViewBlock.jsx

```diff
@@ -6,22 +6,81 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Embed } from 'semantic-ui-react';
-import { isInternalURL, getParentUrl } from '@plone/volto/helpers';
+import {
+  isInternalURL,
+  getParentUrl,
+  flattenToAppURL,
+} from '@plone/volto/helpers';
 import { ConditionalEmbed } from 'volto-gdpr-privacy';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useIntl, defineMessages } from 'react-intl';
 import config from '@plone/volto/registry';
 
+const messages = defineMessages({
+  loadVideo: {
+    id: 'loadVideo',
+    defaultMessage:
+      'Premi Invio per caricare il video, poi premi Tab per navigare sul video.',
+  },
+});
 /**
  * ViewBlock class.
  * @class ViewBlock
  * @extends Component
  */
 const ViewBlock = ({ data, index, isEditMode = false }) => {
+  const intl = useIntl();
+  let placeholder = data.preview_image
+    ? isInternalURL(data.preview_image)
+      ? `${flattenToAppURL(data.preview_image)}/@@images/image`
+      : data.preview_image
+    : null;
+
+  if (!placeholder && data.url) {
+    if (data.url.match('youtu')) {
+      if (!placeholder) {
+        //load video preview image from youtube
+
+        const videoID = data.url.match(/.be\//)
+          ? data.url.match(/^.*\.be\/(.*)/)?.[1]
+          : data.url.match(/^.*\?v=(.*)$/)?.[1];
+        placeholder =
+          'https://img.youtube.com/vi/' + videoID + '/sddefault.jpg';
+      }
+    } else if (data.url.match('vimeo')) {
+      const videoID = data.url.match(/^.*\.com\/(.*)/)[1];
+      placeholder = 'https://vumbnail.com/' + videoID + '.jpg';
+    }
+  }
+
+  const ref = React.createRef();
+  const onKeyDown = (e) => {
+    if (e.nativeEvent.keyCode === 13) {
```

Comment:
Non mi ricordo mai qual è il 13. Sarebbe bello metterlo in un commento, la prima volta che si torna su questo codice.